app/views/schools/advice/electricity_long_term/_comparison.html.erb

```diff
@@ -1,15 +1,26 @@
 <%= render 'schools/advice/section_title', section_id: 'comparison', section_title: t('advice_pages.electricity_long_term.insights.comparison.title') %>
 
-<p>
-  <%= t('advice_pages.electricity_long_term.insights.comparison.how_do_you_compare', school_type: t('common.school_types.' + school.school_type)) %>
-</p>
-
+<% if analysis_dates.one_years_data? %>
+  <p>
+    <%= t('advice_pages.electricity_long_term.insights.comparison.how_do_you_compare',
+          school_type: t('common.school_types.' + school.school_type)) %>
+  </p>
 
-<div class="col">
-  <%= component 'school_comparison', id: 'electricity-comparison', comparison: @benchmarked_usage do |c| %>
-    <% c.with_footer { advice_t('electricity_long_term.insights.comparison.callout_footer') } %>
-  <% end %>
-</div>
+  <div class="col">
+    <%= component 'school_comparison', id: 'electricity-comparison', comparison: @benchmarked_usage do |c| %>
+      <% c.with_footer { advice_t('electricity_long_term.insights.comparison.callout_footer') } %>
+    <% end %>
+  </div>
+<% else %>
+  <p>
+    <%= t('advice_pages.electricity_long_term.insights.comparison.less_than_a_year',
+          available_date: analysis_dates.date_when_one_years_data.to_s(:es_short)) %>
+  </p>
+  <p>
+    <%= t('advice_pages.electricity_long_term.insights.comparison.less_than_a_year_well_managed',
+          usage: format_unit(benchmarked_usage.benchmark_value, :kwh)) %>
```

Comment:
benchmark_value or exemplar_value?

Reply:
Should be `benchmark_value` as these "benchmark schools" are what are referred to as "well managed" on the website.